### PR TITLE
Add web inspector

### PR DESCRIPTION
**Pull request by dustinblackman**
## Overview

Adds ability to enable phantoms web inspector on a specified port.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -70,6 +70,11 @@ var Horseman = function(options) {
 		phantomOptions['cookies-file'] = this.options.cookiesFile;
 	}
 
+	if (this.options.debugPort) {
+		phantomOptions['remote-debugger-port'] = this.options.debugPort;
+		phantomOptions['remote-debugger-autorun'] = 'yes';
```
Comment (awlayton) [inline lib/index.js]: I'm not sure you would *always* want to autorun when using the debugger.

**Comment by awlayton** [comment]
This would resolve #60.

**Comment by dustinblackman** [comment]
@johntitus Bump.

**Comment by awlayton** [comment]
Hey @dustinblackman, if you make a way to not autorun, I would be willing to accept this. I agree autorun should probably be the default when you specify a debug port, but there should be an option for it.

**Comment by dustinblackman** [comment]
@awlayton Sounds good, done!

File: lib/index.js [hunk 2]
```diff
@@ -70,6 +70,13 @@ var Horseman = function(options) {
 		phantomOptions['cookies-file'] = this.options.cookiesFile;
 	}
 
+	if (this.options.debugPort) {
+		phantomOptions['remote-debugger-port'] = this.options.debugPort;
+		if (this.options.debugAutorun !== false) {
+			phantomOptions['remote-debugger-autorun'] = 'yes';
+		}
```
Comment (awlayton) [inline lib/index.js]: You should probably have an `else` here that actually sets the option to `'no'`, rather than relying on it defaulting to `'no'`.

Comment (dustinblackman) [inline lib/index.js]: Done.

**Comment by awlayton** [comment]
Looks good now. Thanks @dustinblackman.

**Pull request merged**